pkg/provider/azure_local_services.go

```diff
@@ -616,3 +616,22 @@ func (az *Cloud) reconcileIPsInLocalServiceBackendPoolsAsync(
 		}
 	}
 }
+
+func isDualStackService(service *v1.Service) bool {
+	return len(service.Spec.IPFamilies) == 2
+}
+
+func (az *Cloud) getBackendPoolNameForCLBService(service *v1.Service) (string, error) {
+	if isDualStackService(service) {
+		return "", fmt.Errorf("dual-stack service is not supported for container load balancer")
```

Comment:
I think we should use names hinting towards "pod ip backend". container load balancer won't mean much here

Reply:
+1